cves/kernel/CVE-2022-3624.yml

```
@@ -1,141 +1,116 @@
-  For multi-line strings, as we do for our instructions, you put a | and then
-  indent by two spaces
-
-  For readability, we hard-wrap multi-line strings at 80 characters. This is
-  not required, but appreciated.
-curated_instructions: |
-  If you are manually editing this file, then you are "curating" it.
-
-  Set the version number that you were given in your instructions.
-
-  This will enable additional editorial checks on this file to make sure you
-  fill everything out properly. If you are a student, we cannot accept your work
-  as finished unless curated is properly updated.
-curation_level: 0
-reported_instructions: |
-  What date was the vulnerability reported to the security team? Look at the
-  security bulletins and bug reports. It is not necessarily the same day that
-  the CVE was created.  Leave blank if no date is given.
-
-  Please enter your date in YYYY-MM-DD format.
-reported_date:
-announced_instructions: |
-  Was there a date that this vulnerability was announced to the world? You can
-  find this in changelogs, blogs, bug reports, or perhaps the CVE date.
-
-  This is not the same as published date in the NVD - that is below.
-
-  Please enter your date in YYYY-MM-DD format.
-announced_date: '2022-10-21'
-published_instructions: |
-  Is there a published fix or patch date for this vulnerability?
-  Please enter your date in YYYY-MM-DD format.
-published_date: '2022-10-21'
-description_instructions: |
-  You can get an initial description from the CVE entry on cve.mitre.org. These
-  descriptions are a fine start, but they can be kind of jargony.
-
-  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
-  read to anyone with some programming experience. We can always pull up the NVD
-  description later to get more technical.
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
-bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
+CWE:
+  - 401
+ipc:
+  note: Vulnerability had to do with failing to release a reference.
+  answer: false
+CVSS: CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L
+bugs: []
+i18n:
+  note: |
+    This vulnerability has nothing to do with internationalization. It is a 
+    memory leak within the application load balancer portion of the Linux kernel.
+  answer: false
+vccs: 
+  commit: d5410ac7b0baeca91cf73ff5241d35998ecc8c9e
+  note: Manually Confirmed
+fixes:
+  commit: [4f5d33f4f798b1c6d92b613f0087f639d9836971]
```

```suggestion
  - commit: 4f5d33f4f798b1c6d92b613f0087f639d9836971
```

commit shouldn't be an array, and it needs to be an array of dictionaries